ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -221,26 +221,6 @@ extension PersonaListView: UITableViewDataSource {
 // MARK: - PersonaListView: UITableViewDelegate
 
 extension PersonaListView: UITableViewDelegate {
-    public func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        guard let section = Section(rawValue: indexPath.section) else {
-            preconditionFailure("heightForRowAtIndexPath: too many sections")
-        }
-
-        switch section {
-        case .personas:
-            return UITableView.automaticDimension
-        case .searchDirectory:
-            switch searchDirectoryState {
-            case .searching:
-                return ActivityIndicatorCell.defaultHeight
```

Comment:
You can delete the definition of this one too since this is the only place it's being used.